example/lib/pages/polygon_custom.dart

```diff
@@ -48,10 +48,11 @@ class TriangleShape extends IntervalShape {
   bool equalTo(Object other) => other is TriangleShape;
 }
 
-List<Figure> simpleTooltip(
-  Offset anchor,
-  List<Tuple> selectedTuples,
-) {
+List<Figure> simpleTooltip({
+  required Offset anchor,
+  required List<Tuple> selectedTuples,
+  required Set<int> selectedIndexes,
+}) {
```

Comment:
I haven't created any example for the selectedIndexes, let me know if you want me to add one.